CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 We welcome contributions to Quickgui.
 
-- Help other Quickgui users by answering questions in the [Quickgui Discussions](https://github.com/quickemu-project/quickgui/discussions) 🛟
+- Help other Quickgui users by answering questions in the [Quickgui Discussions](https://github.com/quickemu-project/discussions/categories/quickgui) 🛟
 - Improve the documentation in [this README](https://github.com/quickemu-project/quickgui/edit/master/README.md) and the [Quickgui Wiki](https://github.com/quickemu-project/quickgui/wiki) 📖
```

Comment:
```suggestion
- Improve the documentation in [this README](https://github.com/quickemu-project/quickgui/edit/master/README.md) and the [Quickemu Wiki](https://github.com/quickemu-project/quickemu/wiki/07-Alternative-frontends) 📖
```
or use the Quickemu wiki ( I'm not a fan of this option, but it at least goes somewhere)